<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsFITS</class>
 <widget class="QWidget" name="OpsFITS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="FITSViewerGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Look &amp;&amp; Feel</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="kcfg_useFITSViewer">
            <property name="toolTip">
             <string>Automatically display received images in the FITS Viewer</string>
            </property>
            <property name="text">
             <string>Use FITS viewer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_singlePreviewFITS">
            <property name="toolTip">
             <string>Display all captured FITS images in a single tab instead of multiple tabs per image.</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Single preview tab</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_singleWindowCapturedFITS">
            <property name="toolTip">
             <string>Display captured FITS images from all cameras in a single FITS Viewer window instead of a dedicated window to each camera.</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Single window capture</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_singleWindowOpenedFITS">
            <property name="toolTip">
             <string>Display opened FITS images in a single FITS Viewer window instead of a dedicated window to each file.</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Single window open</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_focusFITSOnNewImage">
            <property name="toolTip">
             <string>Bring the FITSViewer window to the foreground when receiving a new image</string>
            </property>
            <property name="text">
             <string>Focus on receiving an image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_independentWindowFITS">
            <property name="toolTip">
             <string>Make FITS Viewer window independent from KStars</string>
            </property>
            <property name="text">
             <string>Independent window</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Down Sampling</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="kcfg_AdaptiveSampling">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically down sample images based on available resources.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Adaptive sampling</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="stretchPreviewSamplingLabel">
            <property name="toolTip">
             <string>Set the coarseness of the preview shown when sliding the fitsviewer's stretch parameter sliders. 1 is full resolution, but can be slow, 4 would be coarse resolution and fast.</string>
            </property>
            <property name="text">
             <string>Stretch preview:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="kcfg_StretchPreviewSampling">
            <property name="toolTip">
             <string>Set the coarseness of the preview shown when sliding the fitsviewer's stretch parameter sliders. 1 is full resolution, but can be slow, 4 would be coarse resolution and fast.</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Processing</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="kcfg_AutoStretch">
          <property name="toolTip">
           <string>Always apply auto stretch to images in FITS Viewer</string>
          </property>
          <property name="text">
           <string>Auto stretch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_LimitedResourcesMode">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable limited resource mode to turn off any resource-intensive operations:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Auto debayer&lt;/span&gt;: Bayered images will not be debayered. Only grayscale images are shown.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Auto WCS&lt;/span&gt;: World Coordinate System data will not be processed. WCS maps sky coordinates to image coordinates. Equatorial grid lines, object identification, and telescope slew within an image are disabled.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;3D Cube&lt;/span&gt;: RGB images will not be processed. Only grayscale images are shown.&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Limited resources mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_AutoDebayer">
          <property name="toolTip">
           <string>Automatically debayer captured image if it contains a bayer pattern</string>
          </property>
          <property name="text">
           <string>Auto debayer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_AutoWCS">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically load World-Coordinate-System (WCS) data when opening a FITS file.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning: Only enable this option on very fast machines as it can lead to unpredictable behavior resulting in crashes on slower machines.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Auto WCS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_Auto3DCube">
          <property name="toolTip">
           <string>Process 3D (RGB) FITS images. If unchecked, only first channel is processed.</string>
          </property>
          <property name="text">
           <string>3D cube</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_NonLinearHistogram">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display histogram for the stretched non-linear image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Non linear histogram</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_AutoHFR">
          <property name="toolTip">
           <string>Compute the HFR of images loaded into fitsviewer.</string>
          </property>
          <property name="text">
           <string>Auto compute HFR</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_QuickHFR">
          <property name="toolTip">
           <string>When computing the HFR, run it quickly by only looking at 25% of the image.</string>
          </property>
          <property name="text">
           <string>Quick HFR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="hfrSepLabel">
            <property name="toolTip">
             <string>Set the StellarSolver SEP profile for Capture-image HFR calculations.</string>
            </property>
            <property name="text">
             <string>HFR SEP profile:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="HfrOptionsProfiles"/>
          </item>
          <item>
           <widget class="QPushButton" name="editHfrProfile">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_StellarSolverPartition">
          <property name="toolTip">
            <string>Enable StellarSolver partition. Partitions the image in multiple threads to speed up detecting stars. This may significantly speed up source extraction but may result in unstable operation.</string>
          </property>
          <property name="text">
           <string>StellarSolver Partitioning</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
